Antidote/CallViewController.m

```diff
@@ -0,0 +1,184 @@
+//
+//  CallViewController.m
+//  Antidote
+//
+//  Created by Chuong Vu on 7/1/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "CallViewController.h"
+#import "Masonry.h"
+#import "Helper.h"
+#import "OCTCall.h"
+#import "OCTChat.h"
+#import "OCTSubmanagerCalls.h"
+
+NSString *const kCallViewControllerUserIdentifier = @"callViewController";
+
+@interface CallViewController () <RBQFetchedResultsControllerDelegate>
+
+@property (strong, nonatomic) OCTSubmanagerCalls *manager;
```

Comment:
I think that it would be better not to capture submanager. Controller doesn't control submanager's lifecycle at all. So `weak` would be more preferable.